docs/how-to/grpc-connect.md

```diff
@@ -57,12 +57,12 @@ Next, add a `buf.gen.yaml` in the repository root, containing:
 
 ```
 -- buf.gen.yaml --
-version: v1
+version: v2
 plugins:
-  - plugin: go
+  - local: protoc-gen-go
     out: gen
     opt: paths=source_relative
-  - plugin: connect-go
+  - local: protoc-gen-connect-go
```

Comment:
I believe these are the equivalents in v2 (I just took the same from the example), it doesn't really mention them on https://buf.build/docs/configuration/v2/buf-gen-yaml